### PR TITLE
Store original file size

### LINKED ISSUE #1242: Store original file size during ingestion
It would be helpful to store file size in case something needs to act on it (e.g. sorting, filtering, conditionals).

### PR DESCRIPTION
**Pull request by jimsafley**
fix #1242

### CODE REVIEW

File: application/view/omeka/admin/media/show.phtml [hunk 1]
```diff
@@ -46,6 +46,12 @@ $sectionNavs = [
         <div class="value"><?php echo $escape($media->mediaType()); ?></div>
     </div>
     <?php endif; ?>
+    <?php if (is_numeric($resource->size())): ?>
+    <div class="meta-group">
+        <h4><?php echo $this->translate('Size'); ?></h4>
+        <div class="value"><?php echo sprintf('%s bytes', number_format($resource->size())); ?></div>
```
Comment (zerocrates) [inline application/view/omeka/admin/media/show.phtml]: Possible issue with the use of `number_format` here... we probably should be using the locale-aware numberFormat, but we'd have to add that to our i18n helper to allow it work for people without intl... maybe not worth worrying about too much.

I don't think it should hold up merging but something to look at, possibly file an issue for.

File: application/view/omeka/admin/media/show-details.phtml [hunk 2]
```diff
@@ -25,6 +25,13 @@ $item = $resource->item();
     </div>
     <?php endif; ?>
 
+    <?php if (is_numeric($resource->size())): ?>
+    <div class="meta-group">
+        <h4><?php echo $this->translate('Size'); ?></h4>
+        <div class="value"><?php echo sprintf('%s bytes', number_format($resource->size())); ?></div>
```
Comment (zerocrates) [inline application/view/omeka/admin/media/show-details.phtml]: Needs to be translatable

File: application/view/omeka/admin/media/show-details.phtml [hunk 3]
```diff
@@ -25,6 +25,13 @@ $item = $resource->item();
     </div>
     <?php endif; ?>
 
+    <?php if (is_numeric($resource->size())): ?>
```
Comment (zerocrates) [inline application/view/omeka/admin/media/show-details.phtml]: The value must either be a number or null, right? Seems these should be "not null" checks... is_numeric more appropriate when the value is a number-ish string.

File: application/view/omeka/admin/media/show-details.phtml (hunk 2 above)
Comment (zerocrates) [reply application/view/omeka/admin/media/show-details.phtml]: This is more lines than i intended to grab: the `%s bytes` string specifically.

File: application/view/omeka/admin/media/show-details.phtml (hunk 3 above)
Comment (jimsafley) [reply application/view/omeka/admin/media/show-details.phtml]: Doctrine returns string for bigint data types.

Comment (zerocrates) [reply application/view/omeka/admin/media/show-details.phtml]: ok then

**Pull request merged**